workloads/kube-burner/common.sh

```diff
@@ -242,3 +246,70 @@ prep_networkpolicy_workload() {
   oc apply -f workloads/networkpolicy/clusterrole.yml
   oc apply -f workloads/networkpolicy/clusterrolebinding.yml
 }
+
+churn() {
+  log "Starting to churn workload"
+
+  churn_start=`date +%s`
+  churn_end_time=$((${churn_start} + ${CHURN_DURATION}*60))
+
+  log "Churn duration: ${CHURN_DURATION} minutes"
+  log "Churn wait duration: ${CHURN_WAIT} seconds"
+  log "Churn percentage: ${CHURN_PERCENT}%"
+  log "Churn type: ${CHURN_TYPE}"
+
+  namespace_arry=(`kubectl get namespaces -l kube-burner-uuid=${UUID} --no-headers | awk '{print $1}'`)
```

Comment:
Should be `array`?

Reply:
It can be if you'd like. I just shortened it to save a key stroke :-)

Reply:
@chentex updated to namespace_array